EIPS/eip-6963.md

```diff
@@ -234,11 +235,21 @@ function onPageLoad(): EIP6963ProviderDetail[] => {
 
 ## Security Considerations
 
-The security considerations of [EIP-1193](./eip-1193.md) apply to this EIP.
+### EIP-1193 Security considerations
+
+The security considerations of [EIP-1193](./eip-1193.md) apply to this EIP. Implementers are expected to consider and follow the guidance of the providers they're utilizing as well.
+
+### Prototype Pollution of Wallet Provider objects
+
+Browser extensions, and therefore Wallet extensions, are able to modify the contents of the page and the Provider object by design. Given the provider objects of various Wallets are considered high trusted interface to communicate transaction data. In order to prevent the page or various other extensions from modifying the interaction between the DApp and the Wallet in an unexpected way. The expected way to handle this is to utilize `object.freeze()` on the  `EIP1193Provider` object before the wallet dispatches the `eip6963:announceProvider` Event.
```

Comment:
Might be a wording issue, the second and the third sentences seem unfinished

Reply:
reworded in my other PR

Reply:
rewording suggestion in my other PR